Scripts/Node.cs

```diff
@@ -119,12 +119,12 @@ public enum TypeConstraint {
         protected void OnEnable() {
             if (graphHotfix != null) graph = graphHotfix;
             graphHotfix = null;
-            UpdateStaticPorts();
+            UpdatePorts();
             Init();
         }
 
-        /// <summary> Update static ports to reflect class fields. This happens automatically on enable. </summary>
-        public void UpdateStaticPorts() {
+        /// <summary> Update all ports to reflect class fields. This happens automatically on enable or on redrawing the list of ports. </summary>

```

Comment:
Does not update *all* ports. It updates static ports and dynamic ports managed by DynamicPortList.

Reply:
Noted, completely forgot that non-list-based dynamic ports have to be drawn manually and thus won't be covered by the dynamic port list drawer.

Reply:
Now fixed.